humblerl/wrappers.py

```diff
@@ -29,9 +29,11 @@ def __init__(self, env):
         act_space = self.env.action_space
         if isinstance(act_space, gym.spaces.Discrete):
             self._valid_actions = np.array(list(range(act_space.n)))
-            self._action_space = act_space.n
+            self._action_space = Discrete(num=act_space.n)
         else:
-            raise ValueError("For OpenAI Gym only discrete action space is supported")
+            n_params = len(act_space.low)
+            self._valid_actions = np.array(list(range(n_params)))
```

Comment:
To powinno być identyczne jak `self._action_space`. Tutaj z valid actions chodzi o to, że w danym stanie jakie są dostępne akcje. W przypadku dyskretnych akcji to są po prostu dozwolone indeksy, w przypadku ciągłych powinny być przedziały. A że w OpenAI Gym zawsze wszystkie akcje są dostępne to zwracamy tutaj to samo co `self.action_space`.

Reply:
Przyjrzę się temu, ale to chyba wymaga dodatkowych zmian, np. filtrowanie valid_logits po valid_actions przestanie wtedy działać dla continuous + pewnie parę różnych miejsc. Zobaczę, co da się zrobić.